src/transformers/image_utils.py

```diff
@@ -67,6 +73,17 @@
             PILImageResampling.LANCZOS: InterpolationMode.LANCZOS,
         }
 
+if is_decord_available():
+    from decord import VideoReader, cpu
+
+if is_av_available():
+    import av
+
+if is_cv2_available():
+    import cv2
+
+if is_yt_dlp_available():
+    from yt_dlp import YoutubeDL
```

Comment:
This block breaks lazy importing of `cv2` which `vllm` strictly enforces. It happens when vLLM imports `from transformers.image_utils import ImageInput`. vLLM cannot upgrade to v4.49.0 because of it https://github.com/vllm-project/vllm/pull/13905.

Would it be possible to delay this import? This would be preferable to lazily importing `ImageInput` everywhere it's used in vLLM.